app/allocation/controllers/view.test.js

```diff
@@ -1,3 +1,5 @@
+const { expect } = require('chai')
```

Comment:
Isn't this already available globally?

Reply:
I have no idea how that got there - I certainly didn't do it manually